test/functional/rpc_fundrawtransaction.py

```
@@ -938,6 +941,115 @@ def test_subtract_fee_with_presets(self):
         signedtx = self.nodes[0].signrawtransactionwithwallet(fundedtx['hex'])
         self.nodes[0].sendrawtransaction(signedtx['hex'])
 
+    def test_transaction_too_large(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_transaction_too_large`

Add return type annotation: `None`

(ANN201)